src/interpreter/interpreter.cpp

```diff
@@ -240,14 +249,21 @@ struct ExpressionInterpreter : OverriddenVisitor<ExpressionInterpreter, Flow> {
 
 } // anonymous namespace
 
+Result<> Interpreter::addInstance(std::shared_ptr<Module> wasm) {
+  store.instances.emplace_back(wasm);
+  return Ok{};
+}
+
 std::vector<Literal> Interpreter::run(Expression* root) {
   // Create a fresh store and execution frame, then run the expression to
   // completion.
-  store = WasmStore();
-  store.callStack.emplace_back();
-  store.callStack.back().exprs = ExpressionIterator(root);
+  static Instance dummyInstance = Instance(std::make_shared<Module>());
```

Comment:
It would be good to add a comment explaining that this is a temporary convenience while we are still using the gtests.

Reply:
done